### PR TITLE
Add more spleen items

### PR DESCRIPTION
**Pull request by spaghetti-squash**
#1297

### CODE REVIEW

File: src/diet.ts [hunk 1]
```diff
@@ -316,6 +316,8 @@ function menu(): MenuItem<Note>[] {
    *  > js Item.all().filter((item) => item.fullness > 0 && item.name.indexOf("lasagna") > 0 && getIngredients(item)["savory dry noodles"]).join(", ")
    */
   const lasagnas = $items`fishy fish lasagna, gnat lasagna, long pork lasagna`;
+
```
Comment (docrostov) [inline src/diet.ts:319]: can we add a comment noting what standardSpleenItem means, something like:
``` // standardSpleenItem indicates a spleen item of size 4 with an adventure yield of 5-10. They are functionally equivalent.```

**Review by docrostov — APPROVED** [review]
i would appreciate if you added the comment requested but otherwise it's fine.

are we just ignoring beastly paste? it is ok if the answer is "yeah".

File: src/diet.ts (hunk 1 above)
Comment (horrible-little-slime) [reply src/diet.ts:319]: If you make that a formal suggestion I can commit it right now otherwise it'll have to wait until I'm at my computer

**Comment by horrible-little-slime** [comment]
> i would appreciate if you added the comment requested but otherwise it's fine.
> 
> are we just ignoring beastly paste? it is ok if the answer is "yeah".

Opposite! we already account for beastly paste in our potionmenu

File: src/diet.ts [hunk 2]
```diff
@@ -316,6 +316,12 @@ function menu(): MenuItem<Note>[] {
...
+  /* 
+   * standardSpleenItem indicates a spleen item of size 4 with an adventure yield of 5-10. Taken from the wiki. They are all functionally equivalent.
```
Comment (docrostov) [inline src/diet.ts:321]: wow what an incredible comment whoever wrote this must be a really strong individual with a great spirit and a happy heart, who has never once in their life experienced depression. incredible. great stuff.

**Review by docrostov — APPROVED** [review]
approved as i added the comment

File: src/diet.ts (hunk 2 above)
Comment (docrostov) [reply src/diet.ts:321]: now that i've removed trailing spaces this is even more beautiful

**Review by docrostov — DISMISSED** [review]
crying myself to sleep at how beautiful this comment is

**Pull request merged**